pkg/api/dashboard.go

```diff
@@ -33,12 +34,13 @@ func (hs *HTTPServer) isDashboardStarredByUser(c *models.ReqContext, dashID int6
 		return false, nil
 	}
 
-	query := models.IsStarredByUserQuery{UserId: c.UserId, DashboardId: dashID}
-	if err := hs.SQLStore.IsStarredByUserCtx(c.Req.Context(), &query); err != nil {
+	query := star.IsStarredByUserQuery{UserID: c.UserId, DashboardID: dashID}
+	isStarred, err := hs.dashboardService.DashboardIsStarredByUserCtx(c.Req.Context(), &query)
```

Comment:
I think that we put the return directly here

Reply:
nice catch, you are completely right here